clarity/src/vm/docs/mod.rs

```diff
@@ -2204,6 +2204,28 @@ to deserialize the type, the method returns `none`.
 "#,
 };
 
+const REPLACE_AT: SpecialAPI = SpecialAPI {
+    input_type: "sequence_A, uint, A",
+    output_type: "(response sequence_A uint)",
```

Comment:
I see that the type-checker reports that this returns an optional type instead of a response type.  Can you update the docs accordingly?  Thanks!